lib/order.rb

```diff
@@ -1,18 +1,52 @@
+require "csv"
+require "ap"
+
 module Grocery
   class Order
-    attr_reader :id, :products
+    attr_accessor :id, :products
 
     def initialize(id, products)
       @id = id
       @products = products
     end
 
     def total
-      # TODO: implement total
+      sum =   @products.values.sum
+      return sum + (sum * 0.075).round(2)
+    end
+
+    def add_product (product_name, product_price)
+      if @products.keys.include?(product_name) != true
+        @products[product_name] = product_price
```

Comment:
You don't need the explicit `!= true` in this `if` statement. You could instead say:

```ruby
if !@products.keys.include?(product_name)
# or
unless @products.keys.include?(product_name)
```